dxr/build.py

```diff
@@ -410,15 +411,21 @@ def raise_(exc):
                 if any(fnmatchcase("/" + path.replace(os.sep, "/"), e) for e in ignore_paths):
                     continue  # Ignore the file.
 
-                yield join(root, f)
+                try:
+                    yield join(root, f).decode('utf-8')
+                except UnicodeDecodeError:
+                    continue
```

Comment:
`pass` would suffice here. I misread the `continue` as jumping back to line 394, which had me concerned about skipping the folders[:] thing just below. Oops.